### PR TITLE
AI Control Disabled maphelper

### PR DESCRIPTION
**Pull request by JORJ949**
## About the PR 
Add a maphelper to disable AI control on airlocks and adds it to all rotation maps
Fix https://github.com/goonstation/goonstation/issues/19993
(Also closes both #20015 and #19967)

## Why's this needed? 
I saw #20015 and it gave me a better solution to it and im bored so sprited it and stole @pgmzeta's code for it

### CODE REVIEW

**Review by frawhst — APPROVED** [review]

**Review by pgmzeta — CHANGES_REQUESTED** [review]
good helper

**Review by ZeWaka — CHANGES_REQUESTED** [review]
Review

File: maps/atlas.dmm [hunk 1]
```diff
@@ -20681,11 +20681,9 @@
 /obj/cable{
 	icon_state = "1-2"
 	},
-/obj/machinery/door/airlock/pyro/security{
-	aiControlDisabled = 1;
-	name = "Armory"
-	},
+/obj/machinery/door/airlock/pyro/security,
```
[the remark this answers is not in the document]
Comment (JORJ949) [reply maps/atlas.dmm:20684]: Door names are automatically applied by area name no?

**Comment by JORJ949** [comment]
Am away until friday, feel free to commit any requested changes if you want this merged before then.

File: maps/atlas.dmm (hunk 1 above)
[the remark this answers is not in the document]
Comment (JORJ949) [reply maps/atlas.dmm:20684]: image
Airlocks inherit the name of their area, so there isnt much need to have a varedited name

File: maps/clarion.dmm [hunk 2]
```diff
@@ -19732,15 +19732,14 @@
 	icon_state = "1-2"
 	},
 /obj/machinery/door/airlock/pyro/command{
-	aiControlDisabled = 1;
-	name = "Armory";
 	req_access = null
```
[the remark this answers is not in the document]
Comment (JORJ949) [reply maps/clarion.dmm:19735]: Not something I did, may be worth going through some maps and removing all of these, 
image
donut 3 has 112 "req_access = null" for example. Will fix the armory door ones though.

**Comment by JORJ949** [comment]
I haven't re-added the varedited names to the doors that have them because many maps dont varedit the name because airlocks inherit door name from area, you only need to varedit a door name if you want a name other than the name of the area (e.g. cog2's internal armory doors are "Weapons Storage". Maps could probably use a req_access = null cleanup aswell

**Pull request merged**